src/client/utils/hooks/useSelectedInsuranceTypes.ts

```diff
@@ -13,7 +14,7 @@ export enum InsuranceType {
   DANISH_TRAVEL = 'DANISH_TRAVEL',
 }
 
-const ALL_INSURANCE_TYPES = Object.values(InsuranceType)
+const ALL_INSURANCE_TYPES = Object.values(TypeOfContract)
```

Comment:
How will this work? Because we send in insurance type still.

If we use "type of contract" we should probably rename the hook as well, no?

Reply:
It works because `setSelectedInsuranceTypes` is now called with a `TypeOfContract`, see `src/client/pages/Offer/index.tsx`. But yes, I agree on the name. I wanted to inflict as small a change as possible, but I will look into renaming it 👍

Reply:
Ok, seeing as this could inflict a lot of changes in other parts of the web, I will create a new hook `useSelectedContractTypes` instead and use that in the checkout.

Reply:
Touched a lot of files... https://github.com/HedvigInsurance/web-onboarding/pull/945/commits/5a1391fbc6be7bba174db9e77ae83bf6815ae2ae